JumpscaleLibs/clients/minio/MinioClient.py

```diff
@@ -0,0 +1,114 @@
+"""
+Minio Healing Client
+"""
+
+from Jumpscale import j
+import requests
+
+JSConfigClient = j.baseclasses.object_config
+
+
+class MinioClient(JSConfigClient):
+    """
+    Minio healing client object
+    """
+
+    _SCHEMATEXT = """
+        @url = jumpscale.minio.client
+        name** = "" (S)
+        url** = "" (S)
+        """
+
+    def get_jobs(self):
+        resp = requests.get(f"{self.url}/jobs")
+        resp.raise_for_status()
+        jobs = resp.json()
+    
+        minio_jobs = []
+        for job_id, job in jobs.items():
+            minio_job = MinioJob(
+                self.url, job_id, job["location"], job["type"], job["running"]
+            )
+            minio_jobs.append(minio_job)
+        return minio_jobs
+
+    def heal(self, background=False, file_path=""):
+        url = f"{self.url}/repair"
+        if background:
+            url += "?bg=1"
+            resp = requests.post(url)
+            resp.raise_for_status()
+            return resp.text
+        else:
+            resp = requests.post(url)
+            resp.raise_for_status()
+            return self._read_response_chuncked(resp, file_path)
+    
+    def repair_bucket(self, bucket_name, background=False, file_path=""):
+        url = f"{self.url}/repair/{bucket_name}"
+        if background:
+            url += "?bg=1"
+            resp = requests.post(url)
+            resp.raise_for_status()
+            return resp.text
+        else:
+            resp = requests.post(url)
+            resp.raise_for_status()
+            return self._read_response_chuncked(resp, file_path)
+
+    def repair_bucket_object(self, object_path, background=False, file_path=""):
```

Comment:
I don't like that we have a repair_bucket, then repair_object. etc while it's possible to have all in one single method.

For example 
```python
def heal(report_file=None, dry_run=False, bucket=None, object=None):
```
Note: object can only provided if bucket is provided.

So method gonna be called like this
```python
api = MinioHealClient(...)
api.heal() # full healing of entire instance in he background
api.heal('/path/to/report.txt') # full healing of entire instance but in the foregroun, write report to file
api.heal(report='check.txt', dry_run=True) # full check of the instance
api.heal(bucket='my-bucket') # only heal this bucket in the background
api.heal(bucket='my-bucket', object='path/to/object') # same as above but single object
# you have mix and match flags so you can basically do anything u want.
```